test/evergreen.yml

```diff
@@ -48,12 +48,30 @@ functions:
   "configure wiredtiger": &configure_wiredtiger
     command: shell.exec
     params:
-      working_dir: "wiredtiger/build_posix"
+      working_dir: "wiredtiger"
       shell: bash
       script: |
         set -o errexit
         set -o verbose
-        if [ "$OS" != "Windows_NT" ]; then
+        # Check if the build variant has specified a build type, always default to
+        # autoconf if $is_cmake_build is not declared.
+        if [ ${is_cmake_build|false} = true ]; then
+          if [ "$OS" = "Windows_NT" ]; then
+            # Use the Windows powershell script to configure the CMake build.
+            # We execute it in a powershell environment as its easier to detect and source the Visual Studio
+            # toolchain in a native Windows environment. We can't easily execute the build in a cygwin environment.
+            powershell.exe '.\test\evergreen\build_windows.ps1 -configure 1'
+          else
+            # Compiling with CMake.
+            . test/evergreen/find_cmake.sh
+            mkdir -p cmake_build
+            cd cmake_build
+            $CMAKE ${configure_env_vars|-DCMAKE_TOOLCHAIN_FILE=../build_cmake/toolchains/mongodbtoolchain_v3_gcc.cmake -DCMAKE_C_FLAGS="-ggdb"} \
```

Comment:
The flag settings on this line feel more suitable to be included in the setting for `${posix_configure_flags}`. `${configure_env_vars}` was specifically used to set environment variables that can be leveraged by the configure step.

Reply:
Good point! Thanks for context on `${configure_env_vars}`. I've updated the flags to be under `${posix_configure_flags}`